frontend/.gitignore

```diff
@@ -0,0 +1,27 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
At least some of these appear to be duplicates of the root gitignore file. Suggest sticking all ignores in the root vs spreading across multiple locations.